flexx/ui/examples/code_editor_ace.py

```diff
@@ -0,0 +1,44 @@
+# doc-export: CodeEditor_Ace
+"""
+This example demonstrates a code editor widget based on Ace.
+"""
+
+# todo: Maybe this should be a widget in the library (flexx.ui.Ace) ?
+
+from flexx import ui, app, event
+from flexx.pyscript import window
+
+# Associate Ace's assets with this module so that Flexx will load
+# them when (things from) this module is used.
+base_url = 'https://cdnjs.cloudflare.com/ajax/libs/ace/'
+app.assets.associate_asset(__name__, base_url + '1.2.6/ace.js')
+app.assets.associate_asset(__name__, base_url + '1.2.6/mode-python.js')
+app.assets.associate_asset(__name__, base_url + '1.2.6/theme-solarized_dark.js')
+
+class CodeEditor(ui.Widget):
+    """ A CodeEditor widget based on Ace.
+    """
+    
+    CSS = """
+    .flx-CodeEditor > .ace {
+        width: 100%;
+        height: 100%;
+    }
+    """
+    
+    class JS:
+        
+        def init(self):
+            # https://ace.c9.io/#nav=api
+            self.ace = window.ace.edit(self.node, "editor")
+            self.ace.setValue("import os\n\ndirs = os.walk")
+            self.ace.navigateFileEnd()  # otherwise all lines highlighted
+            self.ace.setTheme("ace/theme/solarized_dark")
+            self.ace.getSession().setMode("ace/mode/python")
+        @event.connect('size')
```

Comment:
While you're making that change, an newline between the two methods here would be nice :)